tests/karate/src/test/java/fe/RollCall/rollCallCreation.feature

```diff
@@ -1,28 +1,42 @@
 @env=android,web
 Feature: Create RollCall
 
-  Background:
+  Scenario: Creating a Roll Call sends right message to backend and displays element
+
+    # Call the lao creation scenario that is front-end agnostic
     * call read('classpath:fe/utils/simpleScenarios.feature@name=create_lao')
+
+    # Call the roll call creation util that is front-end dependant
+    * string rc_name = "Roll-Call"
     * def rc_page_object = 'classpath:fe/utils/<env>.feature@name=create_roll_call'
     * replace rc_page_object.env = karate.env
-    * call read(rc_page_object)
 
-  Scenario: Creating a Roll Call send right message to backend and displays element
-    Given backend.clearBuffer()
-    And backend.setRollCallCreateMode()
+    Given call read(rc_page_object)
+    * backend.clearBuffer()
+    And backend.setValidBroadcastMode()
 
     When click(roll_call_confirm_selector)
 
     # Retrieving sent messages
     * json create_rc_json = buffer.takeTimeout(timeout)
     * string create_rc_string = create_rc_json
+    * print create_rc_string
+
+    # General message verification
+    Then match create_rc_json contains deep { method: 'publish' }
+    * match messageVerification.verifyMessageIdField(create_rc_string) == true
+    And match messageVerification.verifyMessageSignature(create_rc_string) == true
+
+    # Roll Call specific verification
+    And match rollCallVerification.verifyCreateAction(create_rc_string) == true
+    * match rollCallVerification.verifyObject(create_rc_string) == true
```

Comment:
I do not remember if it is easy to do in Karate, but what do you think of a test that looks like that :

```suggestion
    And match getAction(create_rc_string) == CREATE
    * match getObject(create_rc_string) == ROLL_CALL
```

That way, we know exactly what the message should look like, but it is still easy to understand.